docs/ref/models/fields.txt

```diff
@@ -364,6 +364,25 @@ The default value is used when new model instances are created and a value
 isn't provided for the field. When the field is a primary key, the default is
 also used when the field is set to ``None``.
 
+.. versionadded:: 3.1
+
+The Oracle and PostgreSQL database backends support
+:doc:`Query Expressions </ref/models/expressions>` as ``default`` values::
+
+    from django.db import models
+    from django.db.models.functions import ExtractYear, Now, Pi
+
+    class CreatedModel(models.Model)
```

Comment:
Maybe we should name this `ReturningModel` and keep the documentation and test model aligned?